rust-toolchain.toml

```diff
@@ -1,3 +0,0 @@
-[toolchain]
-channel = "1.64.0"
-components = [ "rustfmt", "clippy" ]
```

Comment:
we shouldn't delete the `rust-toolchain.toml` - was it causing issues?

Reply:
I couldn't run any cargo stuff at all with that file present. We should debug that so I can put it back

Reply:
might need a `rustup update` or a `cargo clean` before using it? not sure

Reply:
Yup that needed a clean IIRC

Reply:
`cargo clean` did not fix it. Here's the error message I get when running `cargo fmt`:

`error: the 'cargo-fmt' binary, normally provided by the 'rustfmt' component, is not applicable to the '1.64.0-aarch64-apple-darwin' toolchain`

`cargo clippy` gives something similar. Note that if I change `channel` to "stable" instead of "1.64.0" it works fine 🤔.

Here is some additional data:
```
% cargo --version
cargo 1.64.0 (387270bc7 2022-09-16)
% rustc --version
rustc 1.64.0 (a55dd71d5 2022-09-19)
% rustup component list | rg fmt
rustfmt-aarch64-apple-darwin (installed)
% rustup component list | rg clippy
clippy-aarch64-apple-darwin (installed)
```

Reply:
Uninstalling the toolchain with `rustup toolchain uninstall 1.64.0` and letting it redownload automatically when running `cargo fmt` seems to have fixed it 🤔. Bizarre, but fixed!